datasets/_wikidata/peps/wd_peps.yml

```diff
@@ -46,6 +46,49 @@ data:
   url: https://data.opensanctions.org/contrib/pepdb/holders.csv
   format: CSV
 
+assertions:
+  - metric: entity_count
+    comparison: gt
+    threshold: 160000
+    filter:
+      attribute: schema
+      value: Person
```

Comment:
Looking at this DSL, I totally see why it's very flexible and expressive. But there's a little devil inside of me that's wondering if this isn't a bit smarter than it needs to be, and if we couldn't have something like: 

```yaml
assertions:
  min_entities: 100
  max_entities: 10000
  min_schema:
      Company: 70
      Person: 30
```

This could internally compile down to the same `Assertion` thing, it just feels a little less like it wants to be a framework and more like a human-readable set of constraints....

Reply:
yeah - as a human, parsing 5 lines to decide what the expectation is isn't as much fun as it is for a computer